core/store/migrate/migrations/0191_add_request_id_column_eth_txes_table.sql

```diff
@@ -0,0 +1,7 @@
+-- +goose Up
+
+ALTER TABLE eth_txes ADD COLUMN request_id bytea UNIQUE CHECK (octet_length(request_id) <= 2000);
```

Comment:
Yes that's right. Postgres doesn't treat nulls as equal unless specified so we can have the UNIQUE constraint with multiple null values. Looks like we don't need to separately index the new column either since the UNIQUE constraint automatically takes care of that. https://www.postgresql.org/docs/current/indexes-unique.html